Dockerfile

```diff
@@ -1,33 +1,37 @@
-FROM alpine:3.10
-MAINTAINER Victor Trac <victor@cloudkite.io>
+FROM alpine:3.15.4
+LABEL MAINTAINER Victor Trac <victor@cloudkite.io>
 
-ENV VERSION="1.29.2547.95"
+ENV VERSION="1.30.3116.68"
+
+RUN apk -U upgrade
 
 # Build deps
-RUN apk --no-cache add --update go git bzr wget py2-pip \ 
```

Comment:
I removed `bzr` because I got errors that the package was not available. Online research showed me that it was no longer maintained.